python/tests/test_util.py

```diff
@@ -434,7 +445,25 @@ def test_unicode_table():
 ║1 │ 2│ 3│ 4║
 ╟──┼──┼──┼──╢
 ║5 │ 6│ 7│ 8║
-╟──┼──┼──┼──╢
+╟──┴──┴──┴──╢
+║ rows skipp║
+╟──┬──┬──┬──╢
+║90│10│11│12║
+╚══╧══╧══╧══╝
+"""
+    )
+
+    assert (
+        util.unicode_table(
+            [["1", "2", "3", "4"], ["5", "6", "7", "8"], ["90", "10", "11", "12"]],
+            title="TITLE",
+            row_separator=False,
+        )
+        == """╔═══════════╗
```

Comment:
Nice. Might as well use the ``textwrap.dedent()`` trick here for new code so that we can keep the file indentation looking good.

Reply:
Ah yeah, forgot that trick. Done